experimental/dds/tree2/src/test/feature-libraries/optional-field/optionalField.spec.ts

```diff
@@ -88,33 +89,27 @@ const deltaFromChild2 = ({ change, revision }: TaggedChange<NodeChangeset>): Del
 const tag = mintRevisionTag();
 const change1: TaggedChange<OptionalChangeset> = tagChange(
 	{
-		fieldChange: {
-			id: brand(1),
-			newContent: {
-				set: testTree("tree1"),
-				changes: nodeChange1,
-				buildId: { localId: brand(41) },
-			},
-			wasEmpty: true,
-		},
+		build: [{ id: { localId: brand(41) }, set: testTree("tree1") }],
+		moves: [[{ localId: brand(41) }, "self", "nodeTargeting"]],
+		childChanges: [["self", nodeChange1]],
+		reservedDetachId: { localId: brand(1) },
 	},
 	tag,
 );
 
 const change2: TaggedChange<OptionalChangeset> = tagChange(
-	optionalFieldEditor.set(testTreeCursor("tree2"), false, brand(2), brand(42)),
+	optionalFieldEditor.set(testTreeCursor("tree2"), false, { fill: brand(42), detach: brand(2) }),
 	mintRevisionTag(),
 );
 
 const revertChange2: TaggedChange<OptionalChangeset> = tagChange(
 	{
-		fieldChange: {
-			id: brand(2),
-			newContent: {
-				revert: { revision: change2.revision, localId: brand(2) },
-			},
-			wasEmpty: false,
-		},
+		moves: [
+			[{ localId: brand(2) }, "self", "nodeTargeting"],
+			["self", { localId: brand(42) }, "cellTargeting"],
```

Comment:
I would have expected `["self", { localId: brand(42) }, "nodeTargeting"],` and a new `reservedDetachId`